packages/pancake-uikit/src/widgets/WalletModal/icons/Coin98Wallet.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Svg from "../../../components/Svg/Svg";
+import { SvgProps } from "../../../components/Svg/types";
+
+const Icon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg
+      data-name="Layer 1"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
please remove `xmlns` , it's part of the `Svg` component, and remove `data-name` meta data

Reply:
I've just resolved this. Thank you